cea/visualisation/c_plotter.py

```
@@ -0,0 +1,347 @@
+"""
+PlotManager – Generates the Plotly graph
+
+"""
+
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)